bundles/org.openhab.binding.wiz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -115,7 +115,8 @@
 		<item-type>String</item-type>
 		<label>Light Mode</label>
 		<tags>
-			<tag>Lighting</tag>
+			<tag>Control</tag>
+			<tag>Mode</tag>
```

Comment:
Not sure if it should be mode or light.

Reply:
Again your choice.

Reply:
I just checked the docs and there it actually says that this channel is of type Number, not String...
But as it is technically a String here, I think it should not be `Light`, because it is not clear what sending a `String` value to a Light would actually do. This channel is for choosing presets, so `Mode` might be ok to use.

Reply:
Ok let us keep mode